homeassistant/components/jellyfin/__init__.py

```diff
@@ -15,7 +16,13 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Jellyfin from a config entry."""
     hass.data.setdefault(DOMAIN, {})
 
-    client = create_client()
+    if CONF_DEVICE_ID not in entry.data:
```

Comment:
Its a but confusing to use the Home Assistant constant as its a jellyfin device id AFAICT?

Reply:
It's mostly to identify the client to the server. It shows up under devices in jellyfin admin and it's called such back from the api. would doing CONF_API_DEVICE_ID or CLIENT_DEVICE_ID be better suited?

Reply:
Either of those seem fine to me